raiden/ui/cli.py

```diff
@@ -714,6 +701,22 @@ def run_app(
     except AddressWrongContract:
         handle_contract_wrong_address('secret registry', secret_registry_contract_address)
 
+    database_path = os.path.join(
+        datadir,
+        f'netid_{net_id}',
+        pex(token_network_registry.address),
```

Comment:
Please, be consistent here, with below `address_hex`: `address_hex[:8]` is `0x` + 6 first hex-encoded (checksumed?) address characters, `pex(token_network_registry.address)` is non-`0x`-prefixed, 8 first hex-lowercased registry address.

Reply:
Also, we could include the contract address in the filename, to avoid unnecessarily deep nesting (suggestion, not crucial)